ForumProject/users/admin.py

```diff
@@ -2,8 +2,20 @@
 
 from .models import CustomUser, UserInvestor, UserStartup, UserRoleCompany
 
-# Register your models here.
-admin.site.register(CustomUser)
-admin.site.register(UserInvestor)
-admin.site.register(UserStartup)
-admin.site.register(UserRoleCompany)
+
+class CustomUserAdmin(admin.ModelAdmin):
```

Comment:
Add `list_filter` to improve the ability to filter records based on relevant fields, such as status flags or related model attributes:
```
class CustomUserAdmin(admin.ModelAdmin):
    list_display = ['id', 'email', 'first_name', 'last_name', 'is_active']
    list_editable = ['is_active']
    list_filter = ['is_active', 'email']
```
You should Implement `search_fields` in your admin to allow quick searching, especially useful for user management.
```
class CustomUserAdmin(admin.ModelAdmin):
    search_fields = ['email', 'first_name', 'last_name']
```

Reply:
I slightly improved representation of Users on admin panel by introducing some changes to **users**/admin.py for my personal convenience, for the purposes of managing and tracking the database records during testing / debugging exercise of the Notifications app functionality. Whilst there is indeed a room for perfection of **users**/admin.py, I would be grateful if this pull request could be approved with this, which is out of scope of the current task.